data/plenary/filetypes/builtin.lua

```diff
@@ -0,0 +1,47 @@
+return {
+  extension = {
+    ['fnl'] = 'fennel',
+    ['janet'] = 'janet',
+  },
+  file_name = {
+    ['showing_twitch_chat.kappa'] = 'PogChamp',
+  },
+  shebang = {
+    ['/usr/bin/sh'] = 'sh',
+    ['/usr/bin/bash'] = 'bash',
+    ['/usr/bin/zsh'] = 'zsh',
+
+    ['/bin/sh'] = 'sh',
+    ['/bin/bash'] = 'bash',
+    ['/bin/zsh'] = 'zsh',
+
+    ['/usr/bin/env sh'] = 'sh',
+    ['/usr/bin/env bash'] = 'bash',
+    ['/usr/bin/env zsh'] = 'zsh',
+
+    ['/bin/env sh'] = 'sh',
+    ['/bin/env bash'] = 'bash',
+    ['/bin/env zsh'] = 'zsh',
```

Comment:
If we just save `env sh` or `sh` we could reduce the amount. Because we kinda have to support `/usr/bin` and `/bin`.

Reply:
common_prefixes = {"/usr/bin", "/bin", "/usr/bin/env", ... }
common_fts = {
  { "sh"= "sh"

common_prefix * common_fts